q-pup_config/config/gait/gait.yaml

```diff
@@ -0,0 +1,11 @@
+knee_orientation : ">>"
```

Comment:
smh why does this stupid tool spit out the same thing both here and in the config.json above

Reply:
as far as i could tell from reading the source, it looks to me like the json is only used by setup assistant to save/reload ur inputs and the generated files(ike this yaml) are generated from the json